src/test_report.py

```diff
@@ -27,6 +32,37 @@ def __init__(self, configs, args):
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.db.get_db(self._db_config, api_token)
         self._logger = Logger("config/logger.conf", "test_report")
+        self._email_host = args.email_host
+        self._email_port = args.email_port
+        self._email_send_from = args.email_send_from
+        self._email_subject = args.email_subject
+        self._email_send_to = args.email_send_to
+
+    def create_email(self, email_content, send_to, send_from, email_subject):
+        email_msg = MIMEMultipart()
+        email_text = email.mime.text.MIMEText(email_content, "plain", "utf-8")
+        email_text.replace_header('Content-Transfer-Encoding', 'quopri')
+        email_text.set_payload(email_content, 'utf-8')
+        email_msg.attach(email_text)
+        email_msg['To'] = send_to
+        email_msg['From'] = send_from
+        email_msg['Subject'] = email_subject
+        self._logger.log_message(logging.INFO, email_msg)
+        return email_msg
+
+    def send_email(self, email_msg, email_user, email_password):
+        try:
+            email_server = smtplib.SMTP(self._email_host,
+                                        self._email_port)
+            email_server.starttls()
+            email_server.login(email_user, email_password)
+            email_server.send_message(email_msg)
+        except smtplib.SMTPException as e:
+            self._logger.log_message(logging.INFO, "Check SMTP Exceptions.")
```

Comment:
Having said that `smtplib.SMTPException` is a base class for all SMTP related errors the error message could be a bit more descriptive. Maybe you could make use of the `e` variable that is the actual exception object to get some more data to be displayed?
I'd also changed the log level to `logging.ERROR` as in some cases the `INFO` messages can be suppressed.